yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -156,8 +156,8 @@ export class Sequencer {
       // Only accept processed transactions that are not double-spends,
       // public functions emitting nullifiers would pass earlier check but fail here.
       // Note that we're checking all nullifiers generated in the private execution twice,
-      // we could store the ones already checked and skip them here as an optimization.
-      const processedValidTxs = await this.takeValidTxs(processedTxs);
+      // we could store the ones already checked and skip them here as an optimisation.
```

Comment:
Fighting the good fight!

Reply:
I didn't even know I made that change. :-)